### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by digitaltom**
Manual for package submission

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,4 +32,44 @@ docker run -it yast-rmt-image rspec
...
+##### Factory First
+
+To submit a request to openSUSE Factory, issue this commands in the console:
+
+```bash
+osc sr systemsmanagement:SCC:RMT yast2-rmt openSUSE:Factory --no-cleanup
+```
+
+
+##### Internal Build Service
```
Comment (ikapelyukhin) [inline README.md]: Why would we ever need to submit to IBS? (Considering that it is a linked project from OBS)

File: README.md [hunk 2]
```diff
@@ -32,4 +32,44 @@ docker run -it yast-rmt-image rspec
...
+To make the initial submit for a new SLES version:
+
+```bash
+osc -A https://api.suse.de sr Devel:SCC:RMT yast2-rmt SUSE:SLE-15:GA --no-cleanup
+```
+
+To submit the updated package as a maintenance update to released SLES versions:
+
+```bash
+osc -A https://api.suse.de mr Devel:SCC:RMT yast2-rmt SUSE:SLE-12-SP2:Update --no-cleanup
```
Comment (ikapelyukhin) [inline README.md]: SLE-12? :grimacing:

File: README.md [hunk 3]
```diff
@@ -32,4 +32,44 @@ docker run -it yast-rmt-image rspec
...
+
+To get a maintenance request accepted, each changelog entry needs to reference a bug or feature
+request with `bsc#123` or `fate#123`.
+
+##### Factory First
+
+To submit a request to openSUSE Factory, issue this commands in the console:
+
+```bash
+osc sr systemsmanagement:SCC:RMT yast2-rmt openSUSE:Factory --no-cleanup
```
Comment (ikapelyukhin) [inline README.md]: This project has Yast-related rake tasks, including `rake osc:commit` and `rake osc:sr`. The commit task is pointing to `systemsmanagement:SCC:RMT` project, haven't used the SR task yet.

File: README.md (hunk 2 above)
Comment (digitaltom) [reply README.md]: well, was an example, there is no sle15 update project yet

File: README.md (hunk 3 above)
Comment (digitaltom) [reply README.md]: would it do anything else then `osc sr`?

Comment (ikapelyukhin) [reply README.md]: `osc:commit` runs all kinds of checks (runs specs, check licenses, uncommited files, etc.). I think `osc:sr` will run the same checks, but also issue an SR in the end.

**Comment by digitaltom** [comment]
@ivan I added a not about `rake osc:commit` to the packaging section. 
For the submit request commands, I prefer to stay with the pure `osc` commands, because this is a remote-only operation. 
With `rake osc:sr` it builds locally, commits to the devel project + makes a submit request to Factory. I unintentionally committed this branch to the package by trying the command :-(

**Comment by ikapelyukhin** [comment]
@digitaltom Well, I guess that makes sense that a command to create submission requests actually creates submission requests :D

**Comment by digitaltom** [comment]
@ikapelyukhin submission request i expected, but not commit to the package which cannot get reverted...

**Pull request merged**